indra/newview/skins/default/xui/en/panel_main_inventory.xml

```diff
@@ -217,26 +217,22 @@
        tab_height="30"
        tab_position="top"
        tab_min_width="100"
-       top="0"
-       width="312">
+       top="0">
         <inventory_panel
          bg_opaque_color="DkGray2"
          bg_alpha_color="DkGray2"
          background_visible="true"
          border="false"
          bevel_style="none"
          follows="all"
-         height="338"
          label="MY INVENTORY"
          help_topic="my_inventory_tab"
          layout="topleft"
-         left="0"
          name="All Items"
          sort_order_setting="InventorySortOrder"
          show_item_link_overlays="true"
          preinitialize_views="false"
-         top="16"
-         width="288">
+         scroll.reserve_scroll_corner="false">
```

Comment:
Why did you remove the reservation? Usually we want that avaliable when content can scale both vertically and horizontally (like a very long name)